test/com/esotericsoftware/kryo/FieldSerializerTest.java

```diff
@@ -621,7 +621,71 @@ public void testMultipleTimesAnnotatedMapFields () {
 
 		assertFalse("Exception was expected", true);
 	}
-	
+

```

Comment:
If you don't mind you could put this into a `FieldSerializerInheritanceTest` - sorry that I've suggested to add this to this test, I wasn't aware that it's already that big...